pkgs/operator/handlers.go

```diff
@@ -142,6 +142,28 @@ func (s *Server) reshareHandler(writer http.ResponseWriter, request *http.Reques
 	s.Logger.Info("✅ Stored unsigned reshare message successfully")
 }
 
+func (s *Server) bulkReshareHandler(writer http.ResponseWriter, request *http.Request) {
+	s.Logger.Debug("incoming BULK RESHARE msg")
+	bulkReshareMsg, err := processIncomingRequest(s.Logger, writer, request, wire.BulkReshareMessageType, s.State.OperatorID)
+	if err != nil {
+		s.Logger.Error("Error processing incoming bulk reshare message", zap.Error(err))
+		utils.WriteErrorResponse(s.Logger, writer, err, http.StatusBadRequest)
+		return
+	}
+
+	// hash the message and store it
+	bulkReshare := &wire.BulkReshareMessage{}
+	err = bulkReshare.UnmarshalSSZ(bulkReshareMsg.Message.Data)
+	if err != nil {
+		utils.WriteErrorResponse(s.Logger, writer, err, http.StatusBadRequest)
+		return
+	}
+	hash := eth_crypto.Keccak256(bulkReshareMsg.Message.Data)
```

Comment:
this hash is just for internal storage in map with no security aspect?
we can cosnider a more lightweight function like muhash if so